CHANGES.md

```diff
@@ -3,6 +3,12 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.266
```

Comment:
You're missing:
- [windows: Update certificates](https://github.com/openshift-online/ocm-sdk-go/commit/10df3110f0551292e0a2a3a3f4fd6e7e5350a500)